mayland-blocks/block-templates/index.html

```diff
@@ -9,7 +9,7 @@
 
 <!-- wp:post-excerpt /-->
 
-<!-- wp:post-date /-->
+<!-- wp:template-part {"slug":"post-meta","theme":"a8c-themes/mayland-blocks"} /-->
```

Comment:
The template part doesn't seem to be working unless we add the theme slug, which is very inconvenient